### PR TITLE
Call for early adopters of sel and botan-low

### PR DESCRIPTION
**Pull request by Kleidukos**
cc @jmct @ldillinger please review

### CODE REVIEW

File: content/projects/_index.md [hunk 1]
```diff
@@ -4,9 +4,14 @@ template = "projects/index.html"
 
 [[extra.list]]
 title = "Libsodium bindings"
-content = "Static bindings to the Libsodium C library and Haskell wrapper"
+content = "Bindings to the Libsodium C library and high-level wrapper"
 url = "/projects/libsodium/"
 
+[[extra.list]]
+title = "Botan bindings"
+content = "Bindings to the Botan C++ library and high-level Haskell wrapper"
```
Comment (myShoggoth) [inline content/projects/_index.md:12]: Super nit picky: "high level" makes a bit more sense without the dash.

File: content/blog/early-adopters-call-sel-botan.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+botan-low ^>=0.0.1,
+```
+
+and
+
+```haskell
+sel ^>=0.0.1,
+```
+
+For security reasons, the bindings do not bundle the underlying C and C++ libraries, you have to install them yourself.
```
Comment (ldillinger) [inline content/blog/early-adopters-call-sel-botan.md:33]: I've observed that sometimes people need help finding the installation instructions so I would add something along the lines of 'Please see your specific library's installation instructions for more details.' and link to build instructions directly as well eg https://botan.randombit.net/handbook/building.html and https://libsodium.gitbook.io/doc/installation

Comment (Kleidukos) [reply content/blog/early-adopters-call-sel-botan.md:33]: hmm yeah especially for botan3.

**Comment by jmct** [comment]
This looks good to me!

**Comment by Kleidukos** [comment]
Thanks everyone for the review, I'm pulling the lever. :)

**Comment by Kleidukos** [comment]
Deployment pipeline: https://github.com/haskell-cryptography/haskell-cryptography.github.io/actions/runs/8680568741

**Pull request merged**